src/components/diamond.jsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const DiamondAngle = '26deg';
```

Comment:
also gonna to add using angle parameter instead of this, a bit later in the same PR